website/homepage/src/docs/template.html

```diff
@@ -96,6 +100,21 @@ <h3 class="text-3xl font-light my-4" id="block-layouts">Block Layouts</h3>
             {% endwith %}
           {% endfor %}
         </section>
+        <section id="component-helpers" class="pt-2 flex flex-col gap-10">
```

Comment:
Given that `Examples` are not technically block components I put them in another section. They will be joined shortly by the Reset/Clear button #1719